app.py

```diff
@@ -74,3 +73,9 @@ def index():
         logger.critical("Caught unhandled exception.", exc_info=True)
 
     return ""
+
+
+@app.schedule("rate(1 day)")
```

Comment:
Do we know at what time this will run, or is it random? I suspect it means once a day?

Reply:
Also, this will mean that the lambda will run once a day for dev and prod?

Reply:
This means one a day with no guaranteed time. It will start the lambda (or rather the "other"/new lambda that contains this job) one a day in dev and prod. The cost should be minimal for starting the lambda.